docs/get-started/installation.md

```diff
@@ -1,80 +1,83 @@
+  01. The **Android SDK Location** field should point to a default of _C:\Program Files (x86)\Android\android-sdk_. It probably has a red `X` indicating it's not configured correctly. This is normal.
+  01. Select the **OK** button to close the window.
 
       :::image type="content" source="installation-images/vs-android-settings.png" alt-text="Visual Studio android settings for .NET MAUI.":::
-  
-  Visual Studio should now be able to load the Android SDK Manager. The next step is to install the Android 31 SDK:
 
-  01. In the main Visual Studio window, use the menu to choose **Tools** > **Android** > **Android SDK Manager...**
-  01. In the Android SDK Manager, check the **Android 12.0 API Level 31** item.
-  01. Select the **Apply Changes** button.
+  01. Visual Studio will now attempt to load the Android SDKs. This may involve accepting the Android SDK License:
+
+      :::image type="content" source="installation-images/vs-android-sdk-license.png" alt-text="Visual Studio android SDK license.":::
+
+      If you lack a previous Android install, Visual Studio will download the Android tooling required for you to proceed.
+
+  Visual Studio should now be able to load the Android SDK Manager. The next step is to install the Android 12 SDK:
+
+  01. In the main Visual Studio window, choose the **Tools** > **Android** > **Android SDK Manager...** menu item.
+  01. In the Android SDK Manager, check the **Android 12.0 API Level 31** item and click the **Apply Changes** button.
 
       :::image type="content" source="installation-images/vs-android-sdk.png" alt-text="Visual Studio android SDK manager window for .NET MAUI.":::
 
-For these changes to take effect, you may need to restart Visual Studio.
+  Visual Studio will then install the Android 12 SDK.
+
+## iOS
+
+To build and debug .NET 6 iOS apps from Visual Studio 2022 you must also install the .NET MAUI workload on your Mac build host. On your Mac, in the .NET CLI, run the following command to install the .NET MAUI workload:
+
+```dotnetcli
+dotnet workload install maui
+```
+
+In addition, you must also install the latest beta of Xcode 13 on your Mac.
 
 ## Windows (WinUI 3)
 
 To create apps that target Windows UI Library (WinUI) 3, install the following Visual Studio extension:
 
 - [Single-project MSIX Packaging Tools](https://marketplace.visualstudio.com/items?itemName=ProjectReunion.MicrosoftSingleProjectMSIXPackagingToolsDev17)
 
-  For more information about the required workloads and components for WinUI 3 development, see [Required workloads and components](/windows/apps/project-reunion/set-up-your-development-environment#required-workloads-and-components).
-
 > [!IMPORTANT]
 > You **must** uncomment the Windows `TargetFrameworks` from your _.csproj_ file in order to enable windows development.
 
-## Blazor and WebView
-
 To use the `WebView` or `BlazorWebView` controls on Windows, install the **WebView2 Runtime**:
```

Comment:
Can we remove this as well and add a sense to continue to first app. See dotnet/docs-maui#170.